.github/update-sync-param-files.py

```diff
@@ -13,6 +13,9 @@
 REPO_NAME = "autowarefoundation/autoware.universe"
 REPO_URL = f"https://github.com/{REPO_NAME}.git"
 CLONE_PATH = Path("/tmp/autoware.universe")
+COMMENTS = "\
+  # This file is automatically generated by `update-sync-param-files` workflow, \n\
+  # and can also be changed manually depending on parameter files structure in your custom repository.\n"
```

Comment:
I guess people can't understand what you mean by `can also be changed manually depending on parameter files structure in your custom repository.`.

Instead, how about linking https://github.com/autowarefoundation/autoware-documentation/pull/171 after it's merged?

Reply:
OK, that's a good idea. 
I will make this PR a draft until it is merged